drivers/bus/acpi/compbatt/compbatt.h

```diff
@@ -1,156 +1,325 @@
 /*
- * PROJECT:         ReactOS Composite Battery Driver
- * LICENSE:         BSD - See COPYING.ARM in the top level directory
- * FILE:            boot/drivers/bus/acpi/compbatt/compbatt.h
- * PURPOSE:         Main Header File
- * PROGRAMMERS:     ReactOS Portable Systems Group
+ * PROJECT:     ReactOS Composite Battery Driver
+ * LICENSE:     MIT (https://spdx.org/licenses/MIT)
+ * PURPOSE:     Composite battery main header file
+ * COPYRIGHT:   Copyright 2010 ReactOS Portable Systems Group <ros.arm@reactos.org>
+ *              Copyright 2024 George Bișoc <george.bisoc@reactos.org>
  */
 
+/* INCLUDES *******************************************************************/
+
 #ifndef _COMPBATT_PCH_
 #define _COMPBATT_PCH_
 
 #include <wdm.h>
 #include <batclass.h>
 
+/* DEFINES ********************************************************************/
+
+//
+// I/O remove lock allocate tag
+//
+#define COMPBATT_TAG                            'aBoC'
+
+//
+// Composite battery flags
+//
 #define COMPBATT_BATTERY_INFORMATION_PRESENT    0x04
+#define COMPBATT_STATUS_NOTIFY_SET              0x10
 #define COMPBATT_TAG_ASSIGNED                   0x80
 
+//
+// IRP complete worker mode states
+//
+#define COMPBATT_QUERY_TAG                      1
+#define COMPBATT_READ_STATUS                    2
+
+//
+// Low/High capacity wait constants
+//
+#define COMPBATT_WAIT_MIN_LOW_CAPACITY          0
+#define COMPBATT_WAIT_MAX_HIGH_CAPACITY         0x7FFFFFFF
```

Comment:
:eyes: so values that are >= 0x80000000 aren't supported?

Reply:
That's the the maximum constant set up by the ACPI design standards for properties like power capacities, values, etc. 

Read the **Battery Remaining Capacity** part in the battery status section -> https://uefi.org/htmlspecs/ACPI_Spec_6_4_html/10_Power_Source_and_Power_Meter_Devices/Power_Source_and_Power_Meter_Devices.html#bst-return-package-values

Reply:
Ok thanks for the reference.